lib/content/utilities.ex

```diff
@@ -54,6 +54,7 @@ defmodule Content.Utilities do
 
   def destination_for_prediction(_, 1, "70209"), do: {:ok, :lechmere}
```

Comment:
Ah, good catch! `70503` is one of the _old_ Lechmere stop IDs. With GLX the station was rebuilt nearby and so got new stop IDs (the ones in the API).

So here, the options are: Update the stop ID to the new one, or remove this line entirely. The difference is how signs should behave if for some reason we make predictions for a trip where those predictions end at Lechmere. One reason why that might happen is if there's bus shuttling between Lechmere and Union Sq. If we list the new Lechmere stop ID then the signs will show "Lechmere 5 min" (or whatever), but if we keep it omitted the signs  will say "Union Sq 5 min".

We handle it both ways in other cases: An eastbound (from the branches) Green-E train terminating for some reason at Park St rather than North Station, will show up on signs as "Park St  X min", since Park St is a known station and a common terminus for other lines. However, a westbound (from the trunk) Green-E train terminating at, say, Brigham Circle, will show up on signs as "Heath St   X min". We do that when the stations are notable enough, with the idea that if passengers are trying to get to Heath St, they might be looking for a "Heath St" train (and then will face an unexpected early stop and shuttle, but will still get there) vs not knowing that Brigham Circle is even on the E branch.

My personal opinion is that now that Lechmere is no longer a terminal with a yard, we don't expect trains to ever terminate at Lechmere during normal service, and so it's more in the latter camp, and should show up on signs as "Union Sq", implying that this line should simply be deleted.

Reply:
Agreed, I think it's probably best to remove Lechmere as a terminal. Thank you for catching this @hannahpurcell !